apple/RNSVGUIKit.macos.m

```diff
@@ -0,0 +1,90 @@
+#if TARGET_OS_OSX
```

Comment:
Instead of the entire file being in an if def can we just only build it for the macOS target?

Reply:
We already exclude *.macos.m files on iOS as per the podspec file. I just threw the ifdef in as a "just in case" sort of thing, but yeah, we technically don't need it.

Reply:
Can we delete it if we don't need it? If the code has no function then let's keep things clean and not have it

Reply:
Same request for the iOS file

Reply:
Already done with this commit:
https://github.com/react-native-svg/react-native-svg/pull/1494/commits/a35679a44db0982415b2390ed1e6bf8ab0541bc1